src/PageLayout/PageLayout.tsx

```diff
@@ -0,0 +1,363 @@
+import React from 'react'
+import {Box} from '..'
+import {BetterSystemStyleObject, merge, SxProp} from '../sx'
+
+const REGION_ORDER = {
```

Comment:
Nitpicking: The uppercasing here feels a bit heavy given it's already private and immutable. Should this be an enum instead? 

```js

enum RegionOrder {
  Header,
  PageStart,
  Content
  PaneEnd
 ...
} 

```